include/libdnf/rpm/package_sack.hpp

```diff
@@ -98,27 +98,39 @@ class PackageSack {
         }
     };
 
-    // LoadRepoFlags:
-    // @NONE:                    Load only main solvables
-    // @USE_FILELISTS:           Add the filelists metadata
-    // @USE_PRESTO:              Add presto deltas metadata
-    // @USE_UPDATEINFO:          Add updateinfo metadata
-    // @USE_OTHER:               Add other metadata
     enum class LoadRepoFlags {
-        NONE = 0,
-        USE_FILELISTS = 1 << 1,
-        USE_PRESTO = 1 << 2,
-        USE_UPDATEINFO = 1 << 3,
-        USE_OTHER = 1 << 4,
+        /// Load the primary repodata
+        PRIMARY = 1 << 1,
```

Comment:
Is it necessary to space the items out like this? Would e.g. even this work?
```
PRIMARY = 1 << 1,  /// Load primary repodata
FILELISTS = 1 << 2,  /// Load file lists
...
```

Reply:
Good point.
It makes sense **not** to load the repodata (incl. PRIMARY) when loading only COMPS for example.
Loading FILELISTS and OTHERS without PRIMARY doesn't make sense - that's why changed the bit mask to load also PRIMARY along with these.

Reply:
Well, I wasn't even commenting on that :grin: but since you mention it, I don't think it's a good idea to set more than one bit in a bitmask enum. Imagine someone doing:
```C
LoadRepoFlags flags = LoadRepoFlags::ALL;
flags &= ~LoadRepoFlags::OTHER;  // subtract OTHER from the full set
```

Won't really do what the user would expect...

What I was actually commenting on is just the spacing between the values. I wanted to put the comment and the value on a single line to compact the style.

Reply:
Fixed.